packages/api-explorer/src/index.jsx

```diff
@@ -210,13 +217,14 @@ class ApiExplorer extends React.Component {
                       accordion
                       onChange={this.props.onDocChange}
                     >
-                      {this.props.docs.map((doc, index) => (
-                        <Panel 
+                      {this.props.docs.map(doc => (
+                        <Panel
                           header={this.renderHeaderPanel(doc)}
+                          key={doc._id}
```

Comment:
doc always contains _id field?

Reply:
I don't think doc will ever have `_id` since they're surely not coming from MongoDB, what is this `_id` about?